Code/Engine/Foundation/Strings/HashedString.h

```diff
@@ -167,7 +163,7 @@ class EZ_FOUNDATION_DLL ezTempHashedString
 
   /// \brief Creates an ezTempHashedString object from the given string. Computes the hash of the given string during runtime, which might
   /// be slow.
-  ezTempHashedString(ezStringView sString); // [tested]
+  explicit ezTempHashedString(ezStringView sString); // [tested]
```

Comment:
This was an oversight, that became apparent when I compiled with C++ 20 enabled.

Reply:
Creating a TempHashedString isn't free (because of the hashing). Just implicitly converting to it shouldn't happen, because sometimes you may have other options (or you could just store a TempHashedString to begin with). So I think people should be aware of this.